src/solc_0.8/polygon/child/land/PolygonLandBaseTokenV2.sol

```diff
@@ -29,18 +29,6 @@ abstract contract PolygonLandBaseTokenV2 is IPolygonLand, Initializable, ERC721B
         uint256 size;
     }
 
-    modifier validQuad(
-        uint256 size,
-        uint256 x,
-        uint256 y
-    ) {
-        require(size == 1 || size == 3 || size == 6 || size == 12 || size == 24, "Invalid size");
-        require(x % size == 0 && y % size == 0, "Invalid coordinates");
-        require(x <= GRID_SIZE - size && y <= GRID_SIZE - size, "Out of bounds");
-
-        _;
-    }
-
```

Comment:
why did we move away from using a modifier ?

Reply:
Because it using a function instead of a modifier reduced code size of polygon Land.